src/api/Api.js

```diff
@@ -357,7 +370,9 @@ export default class Project extends EventEmitter {
     }
 
     if (params.query) {
-      searchParams.set("q", params.query);
+      //checking BLOCK_SEARCH_TERMS
+      if (this.searchTermFilteringBlacklist(params.query)) searchParams.set("q", "");
```

Comment:
What does setting `q` to an empty string do? Does that return an empty set of results back to the user?

Reply:
run Spoke and see what happens in the request

Reply:
@igorgark1593 Or you can just tell me :)